Cargo.toml

```diff
@@ -31,6 +31,11 @@ phf = "0.7"
 version = "0.6"
 default-features = false
 
+[dependencies.multipart]
+version = "0.2"
```

Comment:
If you want to tie to a specific version, I recommend `0.2.3`. The previous versions had a lot of bugs.